__snapshots__/index.test.js.snap

```diff
@@ -1318,22 +1318,88 @@ exports[`should pass on packages/eslint-config-base/test/samples/es2015-esm/cont
 Array [
   Object {
     "column": 1,
-    "line": 5,
+    "line": 3,
+    "rule": "lines-around-directive",
```

Comment:
did we change this rule? https://eslint.org/docs/rules/lines-around-directive
It seems to be deprecated in favor of https://eslint.org/docs/rules/padding-line-between-statements

Reply:
We are using both actually, should we change this?

Reply:
We may remove lines-around-directive, but be sure it doesn’t cause troubles by checking snapshots.

Reply:
Actually they are different, we must keep both.

Reply:
See my comment below.